application/features/email.feature

```diff
@@ -205,11 +204,38 @@ Feature: Email
       | to send            | does exist     | does NOT exist  | delete_mfa   | should NOT  |
       | to send            | does NOT exist | does exist      | delete_mfa   | should NOT  |
       | to send            | does exist     | does exist      | delete_mfa   | should NOT  |
-      | to send            | does NOT exist | does NOT exist  | delete_mfa   | should      |
       | NOT to send        | does NOT exist | does NOT exist  | delete_mfa   | should NOT  |
       | to send            | does NOT exist | does NOT exist  | verify_mfa   | should NOT  |
       | NOT to send        | does NOT exist | does NOT exist  | verify_mfa   | should NOT  |
 
+
+  Scenario Outline: When to send an email after a verified u2f mfa option has been deleted.
+    Given we are configured to send mfa disabled emails
+      And we are configured to send mfa option removed emails
+      And no mfas exist
+      And a user already exists
+      And a verified u2f mfa option used to exist
+      And the latest mfa event type was delete_mfa
+    When I check if a mfa <optionRemovedOrDisabled> email should be sent
+    Then I see that a mfa <optionRemovedOrDisabled> email <shouldOrNot> be sent
+
+    Examples:
+      | optionRemovedOrDisabled | shouldOrNot |
+      | option removed          | should NOT  |
+      | disabled                | should      |
+
+  Scenario: When to send an email after an unverified u2f mfa option has been deleted.
+    Given we are configured to send mfa disabled emails
```

Comment:
Whether we are configured to send MFA-disabled emails seems irrelevant to this test (since the When/Then only care about MFA-option-removed emails), so this line could be removed and the `Given` moved down to the next line.